atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/TransactionScopedCacheImpl.java

```diff
@@ -63,15 +63,13 @@ static TransactionScopedCache create(ValueCacheSnapshot snapshot, CacheMetrics m
     @Override
     public synchronized void write(TableReference tableReference, Map<Cell, byte[]> values) {
```

Comment:
I could make this take `Set<Cell>`; I've left it as-is as that's nicer for the callers, but maybe we should just change that anyway.